porting/index.md

```diff
@@ -156,6 +219,12 @@ Here you need to choose your device `cm_[your device]-userdebug`, example if you
 
 ### Building the system.img and hybris-boot.img
 
+Halium will use the mkbootimg tool for creating the boot image. In most cases it is not on the local harddisk, so it can be built by issuing
```

Comment:
This is only necessary because of https://github.com/Halium/hybris-boot/commit/b4db7b876e97ba24f1be42a00974aed86cd2f824 which should be reverted soon(tm)